ontology/templates/ontology/about.html

```diff
@@ -53,6 +53,6 @@ <h3 id="where-and-how-can-i-contribute">Where and how can I contribute?</h3>
 <p>The OEO development was initiated in the project “SzenarienDB” and is
     augmented in the project “LOD_GEOSS”. The development is community based and
     <a href="https://github.com/OpenEnergyPlatform/ontology">takes place on GitHub</a>. </p>
-<p>A OEO steering committee was created in autumn 2019.</p>
+<p>A  <a role="button" class="btn btn-secondary btn-sm" href={% url 'oeo-s-c'%}>OEO steering committee</a> was created in autumn 2019.</p>
```

Comment:
Why is this a button?

Reply:
atm this is the only link to the oeo-s-c page, I wanted to highlight it with the button

Reply:
I think we should stick with a normal link here

Reply:
okay